HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -36,8 +36,11 @@
         "AZURE_STORAGE_CONNECTION_STRING": "<REPLACE WITH AZURE STORAGE CONNECTION STRING>",
         "STORAGE_CONTAINER_NAME": "housing-repairs-online",
         "GOV_NOTIFY_KEY" : "<REPLACE WITH GOV NOTIFY API URL>",
-        "CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY SMS TEMPLATE ID>",
-        "CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY EMAIL TEMPLATE ID>"
+        "CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID": "8cb2913c-e724-48f5-8c3b-78854b1c086e",
```

Comment:
These template IDs are unique and specific to Lincoln, please replace them with placeholders and move these values to the deployed service.